docs/whatsnew/2.0.rst

```diff
@@ -15,7 +15,8 @@ the 1.3.x series of releases.
 In particular, this release includes:
 
 * `Easier use of efficient bytestring Table columns in Python 3`_
-*
+* A new ``std_ddof`` keyword option was added to
```

Comment:
So picky! 😃  Based on previous releases, it actually goes at the top *and* has its own section.  I added the section.